src/Resources/Resources.Test/ScenarioTests/RoleAssignmentTests.cs

```diff
@@ -159,6 +159,13 @@ public void RaGetByScope()
             TestRunner.RunTestScript("Test-RaGetByScope");
         }
 
+        [Fact(Skip = "Needs recording. PS-VSPrompt link and TestFramework are broken at time of commit")]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void RaGetOnlyByRoleDefinitionName()
+        {
+            TestRunner.RunTestScript("Test-RaGetOnlyByRoleDefinitionName");
```

Comment:
Please do not skip a running test.  Please fix any tests that need to be re-recorded, this is one mechanism we use to protect customers from breaking changes.

Reply:
Un-skipped
recorded new test
re-recorded old tests